php/manageevent.php

```diff
@@ -0,0 +1,75 @@
+<!-- 1. change document title -->
+<!-- 2. link css -->
+<!-- 3. link js -->
+<!-- 4. add main html content -->
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- 1. document title -->
+    <title>Manage Events</title>
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="../css/colormodes.css" />
+    <link rel="stylesheet" href="../css/navbar.css" />
+    <!-- 2. bootstrap example css -->
+    <link rel="stylesheet" href="../assets/bootstrap/examplename/example.css" />
+  </head>
+  <body>
+    <!-- 4. main content start -->
+    <div class="container-fluid">
+      <div class="row justify-content-center">
+        <main class="col-md-9 col-lg-10 px-md-4">
+          <h2 class="my-3 pb-3">Events</h2>
+
+          <a class="btn btn-primary" href="addevent.php">Add event</a>
+
+          <div class="table-responsive">
+            <table class="table table-striped">
+              <thead>
+                <tr>
+                  <th scope="col">Event name</th>
+                  <th scope="col" class="col-2">Participants</th>
+                  <th scope="col" class="col-2">Quota</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
```

Comment:
Please use php and mysql statement to display data. Refer to view all data from sir sufyian or `joinevent.php` that I've completed.